async_test.go

```diff
@@ -136,3 +136,39 @@ func retrieveRows(rows *RowsExtended, ch chan string) {
 	ch <- s
 	close(ch)
 }
+
+func TestLongRunningAsyncQuery(t *testing.T) {
+	db := openDB(t)
+	defer db.Close()
+
+	ctx, _ := WithMultiStatement(context.Background(), 0)
+	query := "CALL SYSTEM$WAIT(50, 'SECONDS');use snowflake_sample_data"
```

Comment:
According to my question above - if we can ensure that query is invoked in async manner, maybe we can make this test shorter? Even now tests take like 30 min to run.